app/posts/digital-public-works-ideas.md

```diff
@@ -0,0 +1,86 @@
+
+### 3. Hosting
+
+This might have been tried once before, with [PaaS then being decommissioned](https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/), but it’s still a good idea.
+
+A rebooted service could be simpler but more opinionated, with the benefit of taking care of more of the low-level issues like operating systems, backups, monitoring and scaling.
+
+It could also support easy hosting of prototypes.
+
+### 4. Certifications
+
+There are lots of services where the outcome is that the user receives a certificate or qualification to do something, whether that’s teaching children in schools or working on gas boilers.
+
+Currently, each of these services needs to devise their own way for users to be able to prove or check these certifications.
+
+A central service could allow users to collect all their certifications on a single account and selectively share these with others, either via a code or some other mechanism.
+
+### 5. Registers
+
+Unlike the previous version, this could be a platform for the many services which need to maintain public information such as registered landlords, childminders or building inspectors.
+
+There would be tools for organisations to maintain the data, as well as ways for the public to browse and search it. 
+
+Large-scale registers, such the company register, would still need their own bespoke service, but for small and medium registers a platform would allow these to be built far more quickly and easily.
+
+### 6. Monthly payments
+
+We have GOV.UK Pay for one-off and recurring payments. A possible future extension of this might be users having a single monthly payment to government which includes all the services or licences they currently subscribe to?
+
+This might make it easier for more services to switch from charging a upfront payment every year or so, allowing users to spread the cost and avoid forgetting to renew.
+
+### 7. Statistics publishing
+
+The Office for National Statistics (ONS) has a whole platform for this, but central government also publish a lot of official stats separately on GOV.UK. Currently these are most published as Excel files, PDFs or in other file formats.
+
+Several departments have looked into building platforms for publishing statistics in a more usable and accessible way, and a shared service could be helpful.
+
+It might make sense for this to start by focusing on statistics which are published monthly or quarterly and looking at ways to improve the automation of these.
+
+### 8. Consultations
+
+Whilst not directly connected to digital services, all government departments and agencies run consultations on proposed policies. The GOV.UK platform lets organisations publish documents for consultation, but they usually use commercial third parties for the consultation itself, where the public can answer questions or upload evidence.
```

Comment:
Or, the existing GOV.‌UK platform could be extended to support the features you’re suggesting. It’s already funded, after all!